backend/manager/modules/vdsbroker/src/test/java/org/ovirt/engine/core/vdsbroker/monitoring/VirtMonitoringStrategyTest.java

```diff
@@ -51,7 +52,7 @@ public void setUp() {
         vdsFromDb.setId(vdsId);
         vdsFromDb.setClusterId(clusterId);
 
-        virtStrategy = spy(new VirtMonitoringStrategy(mockCluster(), mockVdsDao(), mockVmDao(), null));
+        virtStrategy = spy(new VirtMonitoringStrategy(mockCluster(), mockVdsDao(), mockVmDao(), null, mock(ImageTransferDao.class)));
```

Comment:
Suggest to update `testVirtCanMoveToMaintenance()` or add a new test method to reflect the new flow:
Make `imageTransferDao.getByVdsId()` return:
1. An empty list (default) - `canMoveToMaintenance()` should return `true`.
2. List with a single `ImageTransfer` with `FINISHED_SUCCESS` phase - `canMoveToMaintenance()` should return `true`.
3.  Add another `ImageTransfer` with `TRANSFERRING` phase to the above list - `canMoveToMaintenance()` should return `false`.